src/Optimizer/Transformer/ServerSideRendering.php

```diff
@@ -900,7 +900,7 @@ private function extractHeightsAttributeCss(Document $document, Element $element
         // TODO: I'm not sure why I initially added this here, it looks very intentional.
         // However, it doesn't match what the NodeJS version does, which is to add padding-top
         // to the inline style of the element.
-        // $this->customSizerStyles[$document->getElementId($element)] = '';
+        // `$this->customSizerStyles[$document->getElementId($element)] = '';`.
```

Comment:
This is supposed to be commented-out code where I was unsure whether it could be removed (hence the TODO). If PHPCS complains about commented-out code here, use `// phpcs:disable <rule-name>` to disable that rule for this specific line.

Reply:
Updated with `// phpcs:ignore`